README.md

```diff
@@ -38,6 +38,41 @@ Then you can use the atomicWriteFile function that accepts a FilePath and a Byte
 See the
 [Haddock documentation](http://hackage.haskell.org/package/atomic-write).
 
+## Contributors ✨
```

Comment:
@Alex0jk, you can also add a badge on top that will automatically update with the total number of contributors. I think it's on all-contributors documentation, but you can check [this PR](https://github.com/stackbuilders/inflections-hs/pull/73/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R9-R11) from Diego to review how it looks 😬